frontend/src/components/DataCard/index.jsx

```diff
@@ -110,6 +110,29 @@ class DataCard extends React.Component {
                                 </Button>
                             </OverlayTrigger>
                         </ButtonToolbar>
+                        {labels.length > 5 && (
+                            <div className="suggestions dropdown">
+                                <button type="button" className="btn btn-info dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                                    Suggested Labels
```

Comment:
So for this, I was wondering if maybe we should just have a table instead of a dropdown? That way the user could see the top 5 suggestions without clicking on the dropdown, and would then just enter their choice in the regular text box. What do you think?

Reply:
I can put a table in! I had played around with it some, and when you have both the label and the description I felt like the card felt a little cluttered so I had switched to a dropdown to save space. But if you want a regular table, then I can move it to that!